test/bigtest/tests/units/settings/MappingProfileDetails/MappingProfileDetails-test.js

```diff
@@ -0,0 +1,210 @@
+                    enabled: true,
+                    transformation: 'test',
+                    recordType: 'HOLDINGS',
+                  }],
+                  userInfo: {
+                    firstName: 'Donald',
+                    lastName: 'S',
+                    userName: 'system_process',
+                  },
+                  outputFormat: 'MARC',
+                  metadata: {
+                    createdDate: '2018-12-04T01:29:36.531+0000',
+                    updatedDate: '2018-12-04T01:29:36.531+0000',
+                  },
+                }],
+              }}
+              onCancel={noop}
+            />
+          </Router>
+        </Paneset>,
+        translationsProperties,
+      );
+    });
+
+    it('should display mapping profile details', () => {
+      expect(mappingProfileDetails.isPresent).to.be.true;
+    });
+
+    it('should display shared full screen view elements', () => {
+      expect(mappingProfileDetails.fullScreen.isPresent).to.be.true;
+    });
+
+    it('should display correct pane title', () => {
+      expect(mappingProfileDetails.fullScreen.headerTitle).to.equal('AP Holdings 1');
+    });
+
+    it('should display expand all button', () => {
+      expect(mappingProfileDetails.expandAllButton.isPresent).to.be.true;
+    });
+
+    it('should display accordions set', () => {
+      expect(mappingProfileDetails.accordions.isPresent).to.be.true;
+    });
+
+    it('should display summary section fields', () => {
+      expect(mappingProfileDetails.summary.name.isPresent).to.be.true;
+      expect(mappingProfileDetails.summary.description.isPresent).to.be.true;
+      expect(mappingProfileDetails.summary.recordType.isPresent).to.be.true;
+      expect(mappingProfileDetails.summary.outputFormat.isPresent).to.be.true;
+    });
+
+    it('should display correct summary fields labels', () => {
+      expect(mappingProfileDetails.summary.name.label.text).to.equal(commonTranslations.name);
+      expect(mappingProfileDetails.summary.description.label.text).to.equal(translations.description);
+      expect(mappingProfileDetails.summary.recordType.label.text).to.equal(commonTranslations.folioRecordType);
+      expect(mappingProfileDetails.summary.outputFormat.label.text).to.equal(translations.outputFormat);
+    });
+
+    it('should display correct summary fields values', () => {
+      expect(mappingProfileDetails.summary.name.value.text).to.equal('AP Holdings 1');
+      expect(mappingProfileDetails.summary.description.value.text).to.equal('AP Holdings 1 description');
+      expect(mappingProfileDetails.summary.recordType.value.text).to.equal(commonTranslations['recordTypes.holdings']);
+      expect(mappingProfileDetails.summary.outputFormat.value.text).to.equal('MARC');
+    });
+
+    it('should display add transformation button', () => {
+      expect(mappingProfileDetails.transformations.addButton.isPresent).to.be.true;
+    });
+
+    it('should display add transformation button', () => {
```

Comment:
By the way, this is the duplication of the block [here](https://github.com/folio-org/ui-data-export/pull/87/files#diff-88bdedc52041f6c06f92d4a5e631c7acR119);